readme.md

```diff
@@ -1,12 +1,30 @@
-[Octopus Deploy][1] is an automated deployment server for professional .NET developers. 
+This repository contains the command line tool (`Octo.exe`) and .NET Client Library (`Octopus.Client`) for [Octopus Deploy][1], an automated deployment server for professional .NET developers. You can use them to create and deploy releases, create and push packages, and manage environments.
 
-**`Octo.exe`** is a command line tool for automating Octopus. You can use it to create and deploy releases, create and push packages, and manage environments.
+`Octo.exe` can be [downloaded from the Octopus downloads page][2], while `Octopus.Client` is [available on nuget.org][3]
 
-The tool can be [downloaded from the Octopus downloads page][2]. 
+## Documentation
+- [Octo.exe][4]
+- [Octopus.Client][5]
 
-Documentation can be found in the [Octopus Deploy documentation](http://docs.octopusdeploy.com/display/OD/Octo.exe+Command+Line). 
+## Issues
+Please see [Contributing](CONTRIBUTING.md)
 
-**Please raise and track issues for this project [here](https://github.com/OctopusDeploy/Issues/issues/).**
+## Compatibility
+Version 3.3 and later of the clients are backwards compatible with Octopus 3.0 or later. Obviously the available features and APIs are limited to those supported by server.
```

Comment:
Hmmm, I find this a bit confusing, because I thought 3.2 of the client was also backwards compatible etc...